plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerColumnHandle.java

```diff
@@ -15,21 +15,49 @@
 package io.trino.plugin.faker;
 
 import io.trino.spi.connector.ColumnHandle;
+import io.trino.spi.predicate.Range;
 import io.trino.spi.type.Type;
 
+import java.util.List;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
 import static java.util.Objects.requireNonNull;
 
 public record FakerColumnHandle(
         int columnIndex,
         String name,
         Type type,
         double nullProbability,
-        String generator)
+        String generator,
+        String min,
```

Comment:
Can you use Optionals please

Reply:
Why? We handle all cases when the value is null.